ci-operator/step-registry/ipi/conf/aws/launcher-info/ipi-conf-aws-launcher-info-commands.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+#set -o nounset
+#set -o errexit
+#set -o pipefail
+
+set +e
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+cp ${CLUSTER_PROFILE_DIR}/pull-secret /tmp/pull-secret
+oc registry login --to /tmp/pull-secret
+ocp_version=$(oc adm release info --registry-config /tmp/pull-secret ${RELEASE_IMAGE_LATEST} --output=json | jq -r '.metadata.version' | cut -d. -f 1,2)
+ocp_major_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $1}' )
+ocp_minor_version=$( echo "${ocp_version}" | awk --field-separator=. '{print $2}' )
+rm /tmp/pull-secret
+
+job_type="${JOB_TYPE:-}"
+user=""
+pull_number=""
+if [[ "${job_type}" == "presubmit" ]]; then
+    user=$(echo "${JOB_SPEC:-}" | jq -r '.refs.pulls[].author')
+    pull_number=${PULL_NUMBER:-unknow}
+fi
+if [[ "${job_type}" == "periodic" ]]; then
+    user="cron"
+fi
+ci_type="prow"
+if [[ "${JOB_NAME_SAFE:-}" == "launch" ]]; then
+    ci_type="cluster-bot"
```

Comment:
should be use `pr-cluster-bot` instead, to differentiate the normal `cluster-bot`?

Reply:
what means `pr-cluster-bot` ?

Reply:
@jianlinliu I'm assuming that `launch` is invoked by the Pull Request, thus the `pr-` prefix.   Where does `JOB_NAME_SAFE` come from?

Reply:
No `launch` means created by robot, `JOB_NAME_SAFE` is from prow ci ENV. For the job launched by pull request, they are indicated by `job_type` as `presubmit`.

Reply:
ok, then leaving it as it's works for me...